<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
     background-color:rgb(220, 220, 220)
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QPushButton {
      border: 2px solid #8f8f91;
      border-radius: 6px;
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }

  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }

  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }

  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QPushButton {
      border: 2px solid #8f8f91;
      border-radius: 6px;
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }

  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }

  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }

  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNAME">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border-bottom-left-radius:3px;
    border-bottom-right-radius:3px;
    border: 1px solid rgb(209 , 209 , 209);
    border-top:transparent;
}
QLineEdit:hover
{
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePASSWD">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border-bottom-left-radius:3px;
    border-bottom-right-radius:3px;
    border: 1px solid rgb(209 , 209 , 209);
    border-top:transparent;
}
QLineEdit:hover
{
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineREPEAT">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border-bottom-left-radius:3px;
    border-bottom-right-radius:3px;
    border: 1px solid rgb(209 , 209 , 209);
    border-top:transparent;
}
QLineEdit:hover
{
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;重复密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/Image/c++father.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
